src/TypeCatalogGen/TypeCatalogGen.cs

```diff
@@ -445,7 +443,8 @@ internal partial class PowerShellAssemblyLoadContext
             StringBuilder sourceCode = new StringBuilder(string.Format(CultureInfo.InvariantCulture, SourceHead, typeNameToAssemblyMap.Count));
```

Comment:
We could set init size.

Reply:
I don't really care about the efficiency here. This executes only at build time.